<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>865</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="wargear_table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>641</width>
      <height>121</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="columnCount">
     <number>7</number>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Range</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>S</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>AP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Description</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="wargear_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Wargear</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="abilities_text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>680</y>
      <width>641</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="abilities_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Abilities</string>
    </property>
   </widget>
   <widget class="QToolBox" name="unit_info_toolbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>641</width>
      <height>401</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="datasheetPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>347</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Datasheet Info</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>621</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="unit_name_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Unit Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="unit_name_line"/>
       </item>
       <item row="0" column="2" rowspan="4">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>78</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="power_rating_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Power Rating</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="power_rating_line"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="battlefield_role_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Battlefield Role</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="battlefield_role_line"/>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="experience_points_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Experience Points</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="experience_points_line"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="faction_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Crusade Faction</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="faction_line"/>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="crusade_points_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Crusade Points</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="crusade_points_line"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="unit_keywords_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Selectable Keywords</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="keywords_line"/>
       </item>
       <item row="4" column="0" colspan="5">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="unit_type_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Unit Type</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="unit_type_line"/>
       </item>
       <item row="5" column="2" rowspan="5">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>98</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="3" colspan="2">
        <widget class="QLabel" name="unit_type_label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Other Selectable Traits</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="equipment_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Equipment</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="equipment_line"/>
       </item>
       <item row="6" column="3" rowspan="4" colspan="2">
        <widget class="QTextEdit" name="selectable_traits_text"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="psychic_powers_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Psychic Powers</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="psychic_powers_line"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="warlord_traits_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Warlord Traits</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="warlord_trait_list"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="relics_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Relics</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="relic_line"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="crusadePage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>639</width>
       <height>345</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Crusade Info</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>621</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="crusade_grid_layout">
       <item row="0" column="0">
        <widget class="QLabel" name="battles_played_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Battles Played</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="battles_played_line"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="battles_survived_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Battles Survived</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="battles_survived_line"/>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="unit_rank_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Unit Rank</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="unit_rank_line"/>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="enemy_unit_total_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Total Enemy Units Destroyed</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="enemy_unit_last_line"/>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="enemy_unit_total_line"/>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="battles_honors_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Battle Honors</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLabel" name="enemy_unit_psychic_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Enemy units destroyed with psychic powers</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="enemy_psychic_last_line"/>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="enemy_psychic_total_line"/>
       </item>
       <item row="2" column="5" rowspan="3">
        <widget class="QTextEdit" name="battle_honors_text"/>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="enemy_unit_ranged_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Enemy units destroyed with ranged weapons</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="enemy_ranged_last_line"/>
       </item>
       <item row="3" column="4">
        <widget class="QLineEdit" name="enemy_ranged_total_line"/>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="enemy_unit_melee_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Enemy units destroyed with melee weapons</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLineEdit" name="enemy_melee_last_line"/>
       </item>
       <item row="4" column="4">
        <widget class="QLineEdit" name="enemy_melee_total_line"/>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="agenda_1_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Agenda 1 Tally</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QLineEdit" name="agenda_1_last_line"/>
       </item>
       <item row="5" column="5">
        <widget class="QLabel" name="battles_scars_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Battle Scars</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="agenda_2_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Agenda 2 Tally</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QLineEdit" name="agenda_2_last_line"/>
       </item>
       <item row="6" column="5" rowspan="2">
        <widget class="QTextEdit" name="battle_scars_text"/>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="agenda_3_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Agenda 3 Tally</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QLineEdit" name="agenda_3_last_line"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QTableView" name="unit_stats_table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="unit_stats_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Unit Stats</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>810</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>810</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
